### PR TITLE
Add input color strings

### PR DESCRIPTION
**Pull request by iamDecode**

### CODE REVIEW

File: cs.lproj/Localizable.strings [hunk 1]
```diff
@@ -282,6 +282,19 @@
...
 
+"INPUT_COLOR_RED" = "Red"; // Untranslated
+"INPUT_COLOR_ORANGE" = "Orange"; // Untranslated
+"INPUT_COLOR_YELLOW" = "Yellow"; // Untranslated
+"INPUT_COLOR_GREEN" = "Green"; // Untranslated
+"INPUT_COLOR_BLUE" = "Blue"; // Untranslated
+"INPUT_COLOR_PURPLE" = "Purple"; // Untranslated
+"INPUT_COLOR_BROWN" = "Brown"; // Untranslated
+"INPUT_COLOR_BLACK" = "Black"; // Untranslated
+"INPUT_COLOR_WHITE" = "White"; // Untranslated
```
Comment (kylef) [inline cs.lproj/Localizable.strings:293]:
```suggestion
"INPUT_COLOR_RED" = "Červené";
"INPUT_COLOR_ORANGE" = "Orange"; // Untranslated
"INPUT_COLOR_YELLOW" = "Yellow"; // Untranslated
"INPUT_COLOR_GREEN" = "Zelený";
"INPUT_COLOR_BLUE" = "Modrý";
"INPUT_COLOR_PURPLE" = "Nachový";
"INPUT_COLOR_BROWN" = "Hnědý";
"INPUT_COLOR_BLACK" = "Black"; // Untranslated
"INPUT_COLOR_WHITE" = "Bílý";
```

File: es.lproj/Localizable.strings [hunk 2]
```diff
@@ -283,6 +283,19 @@
...
 
+"INPUT_COLOR_RED" = "Red"; // Untranslated
+"INPUT_COLOR_ORANGE" = "Orange"; // Untranslated
+"INPUT_COLOR_YELLOW" = "Yellow"; // Untranslated
+"INPUT_COLOR_GREEN" = "Green"; // Untranslated
+"INPUT_COLOR_BLUE" = "Blue"; // Untranslated
+"INPUT_COLOR_PURPLE" = "Purple"; // Untranslated
+"INPUT_COLOR_BROWN" = "Brown"; // Untranslated
+"INPUT_COLOR_BLACK" = "Black"; // Untranslated
+"INPUT_COLOR_WHITE" = "White"; // Untranslated
```
Comment (kylef) [inline es.lproj/Localizable.strings:294]:
```suggestion
"INPUT_COLOR_RED" = "Rosso";
"INPUT_COLOR_ORANGE" = "Arancia";
"INPUT_COLOR_YELLOW" = "Giallo";
"INPUT_COLOR_GREEN" = "Verde";
"INPUT_COLOR_BLUE" = "Blu";
"INPUT_COLOR_PURPLE" = "Viola";
"INPUT_COLOR_BROWN" = "Marrone";
"INPUT_COLOR_BLACK" = "Black"; // Untranslated
"INPUT_COLOR_WHITE" = "Bianca"; // Untranslated
```

Comment (kylef) [inline es.lproj/Localizable.strings:294]:
```suggestion
"INPUT_COLOR_RED" = "Rosso";
"INPUT_COLOR_ORANGE" = "Arancia";
"INPUT_COLOR_YELLOW" = "Giallo";
"INPUT_COLOR_GREEN" = "Verde";
"INPUT_COLOR_BLUE" = "Blu";
"INPUT_COLOR_PURPLE" = "Viola";
"INPUT_COLOR_BROWN" = "Marrone";
"INPUT_COLOR_BLACK" = "Black"; // Untranslated
"INPUT_COLOR_WHITE" = "Bianca";
```

File: it.lproj/Localizable.strings [hunk 3]
```diff
@@ -282,6 +282,19 @@
...
 
+"INPUT_COLOR_RED" = "Red"; // Untranslated
+"INPUT_COLOR_ORANGE" = "Orange"; // Untranslated
+"INPUT_COLOR_YELLOW" = "Yellow"; // Untranslated
+"INPUT_COLOR_GREEN" = "Green"; // Untranslated
+"INPUT_COLOR_BLUE" = "Blue"; // Untranslated
+"INPUT_COLOR_PURPLE" = "Purple"; // Untranslated
+"INPUT_COLOR_BROWN" = "Brown"; // Untranslated
+"INPUT_COLOR_BLACK" = "Black"; // Untranslated
+"INPUT_COLOR_WHITE" = "White"; // Untranslated
```
Comment (kylef) [inline it.lproj/Localizable.strings:293]:
```suggestion
"INPUT_COLOR_RED" = "Rosso";
"INPUT_COLOR_ORANGE" = "Arancia";
"INPUT_COLOR_YELLOW" = "Giallo";
"INPUT_COLOR_GREEN" = "Verde";
"INPUT_COLOR_BLUE" = "Blu";
"INPUT_COLOR_PURPLE" = "Viola";
"INPUT_COLOR_BROWN" = "Marrone";
"INPUT_COLOR_BLACK" = "Black"; // Untranslated
"INPUT_COLOR_WHITE" = "Bianca";
```

File: ja.lproj/Localizable.strings [hunk 4]
```diff
@@ -282,6 +282,19 @@
...
 
+"INPUT_COLOR_RED" = "Red"; // Untranslated
+"INPUT_COLOR_ORANGE" = "Orange"; // Untranslated
+"INPUT_COLOR_YELLOW" = "Yellow"; // Untranslated
+"INPUT_COLOR_GREEN" = "Green"; // Untranslated
+"INPUT_COLOR_BLUE" = "Blue"; // Untranslated
+"INPUT_COLOR_PURPLE" = "Purple"; // Untranslated
+"INPUT_COLOR_BROWN" = "Brown"; // Untranslated
+"INPUT_COLOR_BLACK" = "Black"; // Untranslated
+"INPUT_COLOR_WHITE" = "White"; // Untranslated
```
Comment (kylef) [inline ja.lproj/Localizable.strings:293]:
```suggestion
"INPUT_COLOR_RED" = "赤";
"INPUT_COLOR_ORANGE" = "オレンジ";
"INPUT_COLOR_YELLOW" = "黄";
"INPUT_COLOR_GREEN" = "緑";
"INPUT_COLOR_BLUE" = "青";
"INPUT_COLOR_PURPLE" = "紫";
"INPUT_COLOR_BROWN" = "褐色";
"INPUT_COLOR_BLACK" = "黒";
"INPUT_COLOR_WHITE" = "白";
```

緑 MIDORI!

Fun fact, oragne is a relatively "new" colour and therefore wasn't a native word for it in the language hence it being way longer than others.

File: en.lproj/Localizable.strings [hunk 5]
```diff
@@ -282,6 +282,19 @@
...
+"INPUT_COLOR_RED" = "Red";
+"INPUT_COLOR_ORANGE" = "Orange";
+"INPUT_COLOR_YELLOW" = "Yellow";
+"INPUT_COLOR_GREEN" = "Green";
+"INPUT_COLOR_BLUE" = "Blue";
+"INPUT_COLOR_PURPLE" = "Purple";
+"INPUT_COLOR_BROWN" = "Brown";
+"INPUT_COLOR_BLACK" = "Black";
+"INPUT_COLOR_WHITE" = "White";
+"INPUT_COLOR_RESET" = "Reset";
```
Comment (kylef) [inline en.lproj/Localizable.strings:294]:
```suggestion
"INPUT_COLOR_RESET" = "Default";
```

I wonder if using "default" would be clearer? This sets the "default" colour

File: ja.lproj/Localizable.strings (hunk 4 above)
Comment (iamDecode) [reply ja.lproj/Localizable.strings:293]: Ah cool :)

File: de.lproj/Localizable.strings [hunk 6]
```diff
@@ -282,6 +282,19 @@
...
+"INPUT_COLOR_ORANGE" = "Orange";
+"INPUT_COLOR_YELLOW" = "Gelb";
+"INPUT_COLOR_GREEN" = "Grün";
+"INPUT_COLOR_BLUE" = "Blau";
+"INPUT_COLOR_PURPLE" = "Violett";
+"INPUT_COLOR_BROWN" = "Braun";
+"INPUT_COLOR_BLACK" = "Schwarz";
+"INPUT_COLOR_WHITE" = "Weiß";
+"INPUT_COLOR_RESET" = "Reset"; // Untranslated
+"INPUT_TEXT_COLOR" = "Textfarbe";
```
Comment (kylef) [inline de.lproj/Localizable.strings:295]:
```suggestion
"INPUT_FOREGROUND_COLOR" = "Textfarbe";
```

**Pull request merged**